c++/examples/readdata.cpp

```diff
@@ -85,6 +85,7 @@ main(void)
              */
             H5std_string order_string;
             H5T_order_t  order = intype.getOrder(order_string);
+            (void)order;
```

Comment:
Same for the two in this file (remove variable and cast call's return value to void)

Reply:
Done.